movie/src/components/Card/Card.css

```diff
@@ -36,10 +35,10 @@ img {
   background-color: rgb(234, 244, 244);
   width: 30px;
   height: 30px;
+  border-radius: 30px;
   display: flex;
   align-items: center;
   justify-content: center;
-  border-radius: 30px;
   margin: 5px;
   right: 2px;
   position: absolute;
```

Comment:
why do need to use `margin` and `right` in the same class?

Reply:
margin: 5px  => refer to margin of moreDetail icon on card.
right: 2px => refer to moreDetail be right on the card.